common/src/main/java/com/wynntils/gui/screens/maps/MainMapScreen.java

```diff
@@ -240,16 +234,7 @@ private void renderPois(PoseStack poseStack, int mouseX, int mouseY) {
 
     @Override
     public boolean mouseClicked(double mouseX, double mouseY, int button) {
-        if (button == GLFW.GLFW_MOUSE_BUTTON_RIGHT) {
```

Comment:
Why do you remove this?

Reply:
its duplicate functionality for centering

Reply:
How is it duplicate? I don't get it. One time we center around the player, one time, the waypoint.